notifications/notifications.go

```diff
@@ -20,8 +21,15 @@ func Send(client Client, endpoint string, msg string, headers map[string]string)
 func ReportSchedule(client Client, schedule *schedule.Schedule) bool {
 	success := true
 	receiver := config.NotificationsAttacks()
-	msg := fmt.Sprintf("{\"text\": \"schedule:\n%s\n\"}", schedule)
+	sched := "schedule"
 
+	kubeMonkeyID := os.Getenv("KUBE_MONKEY_ID")
+
+	if kubeMonkeyID != "" {
+		sched = fmt.Sprintf("schedule(id: %s)", kubeMonkeyID)
```

Comment:
Hmm 🤔 
Should the pattern be configurable, or maybe json friendly by default?
I guess the schedule is more for human consumption and not machine, but it should still be easy to parse.

Reply:
@worldtiki apologies for the late reply. Can you give me an example of what you think that would be appropriate here? I was thinking of having a static string here and make the pattern configurable in a next PR as to not mix too many things in a single PR

Reply:
sorry for the delay.

my comment was if the schedule (or any notification) should have a human format and a machine format.

it may make sense to send this as json so that some upstream system can parse the schedule and do something with that information.
or maybe we just want to send the text in a format that's easy for a person to read.

right now this is a mix of both:

```
{"text": "schedule(id: foo):
	********** Today's schedule **********
	k8 Api Kind    Kind Namespace    Kind Name    Termination Time
	-----------    --------------    ---------    ----------------
	Pod            default              name      03/17/2021 20:50:53 +0000 GMT
	Pod            default              name      03/17/2021 20:50:53 +0000 GMT
	********** End of schedule **********
"}
```

one option is to make this entire thing a valid json document, which maybe is not the best idea right now.
another would be to tidy up this schedule so that it's easier for a human to read. maybe like this?


```
{"text": "
	********** Today's schedule **********
	Cluster k8 Api Kind    Kind Namespace    Kind Name    Termination Time
	-------  -----------    --------------    ---------    ----------------
	foo      Pod            default              name      03/17/2021 20:50:53 +0000 GMT
	foo      Pod            default              name      03/17/2021 20:50:53 +0000 GMT
	********** End of schedule **********
"}
```

Reply:
Sure I will adapt it

Reply:
@worldtiki thank you for the suggestions. I have implemented your suggestion to drop the `schedule(id: foo):` part of the report.

Reply:
![image](https://user-images.githubusercontent.com/1187103/112117554-62ec6b00-8bb3-11eb-9741-ce7d41bba889.png)

The format is still incorrect (indentation) :(

Reply:
@worldtiki you are right for course. I have also fixed the `NoTermination` which was also wrong.